src/components/Search/SearchAutocomplete.tsx

```diff
@@ -53,20 +53,29 @@ const SearchAutocomplete: React.FC<{
       onChange={onSelect}
       getOptionSelected={getOptionSelected}
       filterOptions={createFilterOptions({
-        matchFrom: isZip ? 'any' : 'start',
+        matchFrom: checkForZipcodeMatch ? 'any' : 'start',
         limit: filterLimit,
         stringify: stringifyOption,
       })}
+      popupIcon={<span />} // adding an empty span removes default MUI arrow icon
       renderInput={params => (
         <TextField
           {...params}
           variant="outlined"
-          placeholder="Search for your state, county, or zip"
+          placeholder="Search for a state, county, or zip code"
         />
       )}
       renderOption={option => {
         return <MenuItem region={option} zipCodeInput={zipCodeInput} />;
       }}
+      openOnFocus
+      onOpen={() => {
+        if (setHideMapToggle) setHideMapToggle(true);
```

Comment:
this is very much a tiny nit, but if statements without brackets make me a bit uncomfy.  Do you have a preference for them? if not would advocate for brackets but if we don't have it set in our eslint, I'm not too worried either

Reply:
no preference at all! just added them in